packages/melos/lib/src/commands/version.dart

```diff
@@ -265,6 +265,11 @@ mixin _VersionMixin on _RunMixin {
     );
 
     for (final package in dependentPackagesToVersion) {
+      // If updateDependentsVersions is set to false, we do not perform updates.
+      if (!updateDependentsVersions) {
+        continue;
+      }
```

Comment:
We don't have to go into this loop at all if `updateDependentsVersions` is false right?